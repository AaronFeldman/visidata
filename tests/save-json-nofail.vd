sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test			save-sheet	tests/golden/save-json-nofail.csv	^S	
test			save-sheet	tests/golden/save-json-nofail.json	^S	
test			save-sheet	tests/golden/save-json-nofail.md	^S	
test			save-sheet	tests/golden/save-json-nofail.html	^S	
test			save-sheet	tests/golden/save-json-nofail.txt	^S	
