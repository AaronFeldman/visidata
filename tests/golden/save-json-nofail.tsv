key1	key2	qty	amt
2016-01-01 11:00:00	foo	1	
2016-01-01 1:00		2	3
	baz	4	43.2
{"key1": "2017-12-25 8:43", "foo", "amt": ".03", "qty": null}	{"key1": "2017-12-25 8:43", "foo", "amt": ".03", "qty": null}	{"key1": "2017-12-25 8:43", "foo", "amt": ".03", "qty": null}	{"key1": "2017-12-25 8:43", "foo", "amt": ".03", "qty": null}
2017-12-25 8:44	bar	16	.3
	baz	32	3.3
2018-07-27 4:44		64	9.1
2018-07-27 16:44	bar		
2018-07-27 18:44	baz	256	.01
2018-10-20 18:44	foo	30.0	.01
